### PR TITLE
DOCS-5038 Source Code Integration Edits (Embedding Git Metadata)

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do?

Add a section about embedding git metadata to tag telemetry with stack traces and profiles for a variety of language tracers.

### Motivation

DOCS-5038

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/integrations/guide/source-code-integration.md [hunk 1]
```diff
@@ -31,7 +31,9 @@ Datadog Agent v7.35.0 or later is required.
 
 If you have [APM][6] set up already, navigate to [**Integrations** > **Link Source Code**][7] and configure the source code integration for your backend services.
 
-## Tag your telemetry
+## Link active commits
```
Comment (alai97) [inline content/en/integrations/guide/source-code-integration.md:34]:
```suggestion
## Link active commits

You can link active commits by either [**tagging your telemetry**](#tag-your-telemetry) or [**embedding git information**](#embed-git-information-in-your-artifacts-on-ci) in the deployed artifact. 
```

File: content/en/integrations/guide/source-code-integration.md [hunk 2]
```diff
@@ -96,6 +100,25 @@ export DD_TAGS="git.commit.sha:<FULL_GIT_COMMIT_SHA>,git.repository_url:git-prov
 
 Datadog only captures the repository URL, the commit SHA of the current branch, and a list of tracked file paths—Datadog does not ingest or store any user code.
 
+### Embed git information in your artifacts on CI
+
+You can tag your telemetry with git information, such as the repository URL and commit hash, for artifacts built in a CI pipeline. The [Datadog Tracing Libraries][9] use this information to tag your telemetry.
```
Comment (alai97) [inline content/en/integrations/guide/source-code-integration.md:105]:
```suggestion
You can embed git information such as the repository URL and commit hash in your artifact. The [Datadog Tracing Libraries][9] use this information to automatically link the active commit to your APM service.

Select one of the following languages that supports embedding git information:
```

**Review by vidurkhannadatadog — APPROVED** [review]

File: content/en/integrations/guide/source-code-integration.md [hunk 3]
```diff
@@ -31,7 +31,11 @@ Datadog Agent v7.35.0 or later is required.
 
 If you have [APM][6] set up already, navigate to [**Integrations** > **Link Source Code**][7] and configure the source code integration for your backend services.
 
-## Tag your telemetry
+## Link active commits
+
+You can link active commits by either [**tagging your telemetry**](#tag-your-telemetry) or [**embedding git information**](#embed-git-information-in-your-artifacts-on-ci) in the deployed artifact.
```
[the remark this answers is not in the document]
Comment (drichards-87) [reply content/en/integrations/guide/source-code-integration.md:36]: Just a small suggestion as I'm not sure if this was intentional (bolding the link text).

**Review by drichards-87 — APPROVED** [review]
Left one minor suggestion/question. PR is approved.

**Pull request merged**